package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "appcenter-cli",
+  "name": "@data4life/appcenter-cli",
```

```suggestion
  "name": "@d4l/appcenter-cli",
```

So it's better aligned with the rest of our npm packages https://www.npmjs.com/~d4l-team